irma/1.0.2/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM ubuntu:focal as app
+
+# ARG sets environment variables during the build stage
+# USHER ver is in the conda version style.
+ARG IRMA_VER="1.0.2"
+ARG LABEL_VER="0.6.1"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="IRMA"
+LABEL software.version=$IRMA_VER
+LABEL description="IRMA was designed for the robust assembly, variant calling, and phasing of highly variable RNA viruses. Currently IRMA is deployed with modules for influenza, ebolavirus and coronavirus."
+LABEL website="https://wonder.cdc.gov/amd/flu/irma/"
+LABEL license="https://wonder.cdc.gov/amd/flu/irma/disclaimer.html"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ perl \
+ default-jre \
+ apt-transport-https \
+ ca-certificates \
+ gnupg \
+ libcurl4-openssl-dev \
+ libssl-dev \
+ libxml2 \
+ libxml2-dev \
+ libncurses5 \
+ wget \
+ git
```

Comment:
You could clean up some `apt` garbage by adding this to line 37 & 38. Might save a little bit of space
```
git && \
apt-get autoclean && rm -rf /var/lib/apt/lists/*
```